src/nodes.coffee

```diff
@@ -4469,6 +4498,12 @@ exports.StringWithInterpolations = class StringWithInterpolations extends Base
   constructor: (@body, {@quote, @startQuote} = {}) ->
     super()
 
+  @fromStringLiteral: (stringLiteral) ->
+    updatedString = stringLiteral.withoutQuotesInLocationData()
+    updatedStringValue = new Value(updatedString).withLocationDataFrom updatedString
+    new StringWithInterpolations Block.wrap([updatedStringValue]), quote: stringLiteral.quote
+    .withLocationDataFrom stringLiteral
```

Comment:
Basically, the `StringWithInterpolations`'s location data should include the quotes but the inner "text-chunk" `StringLiteral`'s location data shouldn't include the quotes